webapp/src/js/services/android-api.js

```diff
@@ -167,7 +167,19 @@ angular.module('inboxServices').factory('AndroidApi',
           try {
             MRDT.respond(JSON.parse(response));
           } catch(e) {
-            return $log.error(new Error('Unable to parse JSON response from android app: "' + response + '"'));
+            return $log.error(new Error('Unable to parse JSON response from android app: "' + response + '", error message: "' + e.message + '"'));
```

Comment:
The second parameter to `Error` is a filename, not additonal messages. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error